### ISSUE TITLE
other than built in apps all apps are having small titlebar

### ISSUE THREAD

**Issue by dtbj** [vinceliuice/Orchis-theme#177]
other than built in apps all apps are having small titlebar
SEE in settings
![image](https://user-images.githubusercontent.com/87176273/152679522-51cf0c3e-d688-43af-bd53-ef84c7b37fe0.png)

and now see in vscodium:
it's so small and making me feel bad
![image](https://user-images.githubusercontent.com/87176273/152679561-049cc1c7-6efd-4cd9-ae7f-f23a5313b09a.png)

**Comment by vinceliuice** [comment]
This decoration titlebar only show a title on it, why you want it to be so big?
It's not like CSD titlebar, there are many buttons on it, so it should be a bigger size, but for this, I don't think a big size looks better

**Comment by dtbj** [comment]
@vinceliuice I agree but in default adwaita it's bigger I need it bit bigger I am not asking it to be so big just like settings I have small monitor so I really have problems to click that icon see the freetube title bar it's so small:
![image](https://user-images.githubusercontent.com/87176273/152680585-6f5f5f33-f6e1-4a18-9a87-369c5d7d1172.png)
and in default theme see it's looking good:
![image](https://user-images.githubusercontent.com/87176273/152680606-f09ff2e8-418e-4889-a5db-db8bd118c870.png)

**Comment by vinceliuice** [comment]
OK, a bit bigger maybe not bad, I'll adjust it

**Comment by dtbj** [comment]
@vinceliuice Sir can you please tell me css class I would like to play with it once.

**Comment by vinceliuice** [comment]
It's `.default-decoration` you can change this: https://github.com/vinceliuice/Orchis-theme/blob/e6467a808950b3d2bf6efa86e382f55e9ca6281d/src/_sass/gtk/_common-3.20.scss#L1267

**Comment by dtbj** [comment]
Thank you so much :-)

**Comment by dtbj** [comment]
@vinceliuice It doesn't seems to be working :-(

**Comment by dtbj** [comment]
It worked now I want to make this icons also big 
![image](https://user-images.githubusercontent.com/87176273/152682010-e856298f-aec1-44e8-9370-f27ac10763df.png)
what can I do sorry to disturb you last question?

**Comment by dtbj** [comment]
@vinceliuice please help what's happening I have removed your theme from .themes directory and rebooted also and applied default theme also but I am unable to get rid of this big titlebar 
![image](https://user-images.githubusercontent.com/87176273/152683319-581f7954-014a-498f-a75e-c522af3099be.png)

What's wrong?

**Comment by vinceliuice** [comment]
just 'ctrl + f2', put a 'r' in the box then 'enter' to restart the plasma

**Comment by dtbj** [comment]
Thanks finally it's fixed :-)
